<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="firstImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>869</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>50</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="loadButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>0</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="featureSelect">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>45</y>
       <width>90</width>
       <height>100</height>
      </rect>
     </property>
     <property name="title">
      <string>  Feature </string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QRadioButton" name="siftButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SIFT</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="surfButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>35</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SURF</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="mserButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>MSER</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="harrisButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>HARRIS</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="detectButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>150</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Detect Feature</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
